constraint-solver/src/solver/exhaustive_search.rs

```diff
@@ -69,7 +69,15 @@ pub fn get_brute_force_candidates<'a, T: RuntimeConstant, V: Clone + Hash + Ord>
     rc: impl RangeConstraintProvider<T::FieldType, V> + Clone + 'a,
 ) -> impl Iterator<Item = BTreeSet<V>> + 'a {
     constraint_system
-        .expressions()
+        .algebraic_constraints()
+        .iter()
+        .map(|c| &c.expression)
+        .chain(
+            constraint_system
+                .bus_interactions()
+                .iter()
+                .flat_map(|b| b.fields()),
```

Comment:
I replicated the original behaviour, but I'm wondering if it wouldn't be better to compute the set of variables in the whole bus interaction here instead of in just one field. What do you think, @georgwiese ?